sigopt_sklearn/search.py

```diff
@@ -321,21 +326,17 @@ def _fit(self, X, y, parameter_iterable=None):
                                             self.fit_params,
                                             return_parameters=True,
                                             error_score=self.error_score)
-                        for parameters in parameter_configs
-                        for train, test in cv)
+                        for parameters, train, test in jobs)
             except TimeoutError:
                  obs_timed_out = True
 
             if not obs_timed_out:
                 # grab scores from results
-                n_folds = len(cv)
                 for sidx, suggestion in enumerate(suggestions):
-                    out_sidx = n_folds * sidx
-                    scores = [o[0] for o in out[out_sidx:(out_sidx+n_folds)]]
+                    score = out[sidx][0]
                     self.sigopt_connection.experiments(self.experiment.id).observations().create(
                         suggestion=suggestion.id,
-                        value=numpy.mean(scores),
-                        value_stddev=numpy.std(scores))
```

Comment:
Now that there's one observation per fold, we don't aggregate them here, since that's done internally. `optimizer.py` in sigopt-api aggregates folds using numpy just like what we used to do here.